### DISCUSSION TITLE
Scrollbar color follows OS mode, not Furo theme light/dark mode

### DISCUSSION THREAD

**Discussion by ShawnHymel** [pradyunsg/furo#689] [Potential Issue] [answered]
OS: macOS 13.4.1
Browser: Chrome 115.0.5790.170

The scroll bars on the page are always dark if my OS theme is set to "Dark." As you can see in this screenshot, even the official Furo documentation site has dark scrollbars. They should follow the theme settings and only revert to the OS mode if the "Auto" mode is selected on the site.

![Screenshot 2023-08-09 at 6 10 03 PM](https://github.com/pradyunsg/furo/assets/5232145/28612c9d-11a3-45b2-ab54-b7c2fea43af4)

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/issues/721 filed for tracking this. :)
